tw-NEDa-frontend/src/app/app.module.ts

```diff
@@ -28,10 +36,14 @@ import { EarthquakeDialog } from './components/earthquakes/earthquake.dialog';
 
     MaterialModule.forRoot(),
     FlexLayoutModule,
-    NgxDatatableModule
+    NgxDatatableModule,
+    AgmCoreModule.forRoot({
```

Comment:
Can you add a comment to specify what is this for to share the knowledge across the team?